### ISSUE TITLE
Published package has broken deps

### ISSUE THREAD

**Issue by madtrick** [contentful-labs/keepachangelog#5]
The package depends on `lodash-node` which on the published version is specified as a `devDependency` so it doesn't get installed.

**Comment by madtrick** [comment]
Another broken dependency is `babel-register`. Required in the `index.js` file https://github.com/contentful-labs/keepachangelog/blob/3d9e9a53a2c5f213e4289b75345248f22e095b6d/index.js#L8 but not exposed as a dependency

**Comment by geigerzaehler** [comment]
> The package depends on lodash-node which on the published version is specified as a devDependency so it doesn't get installed.

This has been fixed in 7f117e417ec98a2e8a3e3360efaf5f08a09d6a86. I published the new version to npm.

> Another broken dependency is babel-register

I am not sure if we want to include it. If you install the package from npm there is no need for this since the source has already been compiled with babel. If you use it from the repository, you need to have the dev deps installed.

**Comment by madtrick** [comment]
@geigerzaehler I've just installed the dependency from the npm registry and the issue with `babel/register` persists.

This is the `index.js` file inside the installed package:

``` js
'use strict';

try {
  module.exports = require('./dist');
} catch (e) {
  if (e.code != 'MODULE_NOT_FOUND')
    throw e;
  require('babel/register');
  module.exports = require('./src');
}
```

**Comment by geigerzaehler** [comment]
Just released `v0.2.1`, this should work now.

**Comment by madtrick** [comment]
Yep, thx
